### PR TITLE
Unique open api path parameters

### PR DESCRIPTION
**Pull request by kubinio123**

### CODE REVIEW

File: apispec/openapi-model/src/main/scala/sttp/tapir/openapi/OpenAPI.scala [hunk 1]
```diff
@@ -178,7 +178,7 @@ final case class PathItem(
     patch: Option[Operation] = None,
     trace: Option[Operation] = None,
     servers: List[Server] = List.empty,
-    parameters: List[ReferenceOr[Parameter]] = List.empty,
+    parameters: Set[ReferenceOr[Parameter]] = Set.empty,
```
Comment (adamw) [inline apispec/openapi-model/src/main/scala/sttp/tapir/openapi/OpenAPI.scala:181]: I'm afraid we'll have to de-duplicate in some other way - using a `Set` we might loose the ordering

Comment (kubinio123) [reply apispec/openapi-model/src/main/scala/sttp/tapir/openapi/OpenAPI.scala:181]: Ok, I was not aware that order of parameters is important here, I will add some de-duplication logic to `EndpointToOpenAPIPaths` then

Comment (adamw) [reply apispec/openapi-model/src/main/scala/sttp/tapir/openapi/OpenAPI.scala:181]: Yes it's better to keep things less surprising ;)

**Comment by adamw** [comment]
Ha ended up quite simple :)

**Pull request merged**